flight_planning/v1/telemetry.yaml

```diff
@@ -0,0 +1,220 @@
+info:
+  title: >-
+    Telemetry definitions used in flight planning interface.
+components:
+  schemas:
+    FlightTelemetry:
+      description: >-
+        When this information is present and the USS supports telemetry ingestion for a flight, the USS receiving this information should enqueue these telemetry reports to be delivered to the system, as if they were coming from the user, at (or soon after) the specified times.  If/when the associated flight is canceled or closed, any telemetry not yet delivered to the system should be dequeued and discarded.
+      type: object
+      properties:
+        states:
+          description: >-
+            The set of telemetry data that should be injected into the system (as if reported by the user or the user's system) at the appropriate times (and not before) for this flight.
+          type: array
+          items:
+            $ref: '#/components/schemas/AircraftState'
+    AircraftState:
+      description: State of an aircraft for the purposes of simulating the execution of a flight plan.
+      required:
+        - timestamp
```

Comment:
Is there a reason that ONLY the timestamp is required?  With no other elements, do we really have any state information that is reportable? Only viewing in GitHub review editor... if I missed something in regards to required elements, just LMK.

Reply:
This interface is essentially puppeteering a user of the USS's system.  It seems likely that we may want to simulate loss of one or more telemetry components at some point, but it seems like we would always know when the injected telemetry should be injected into the system under test.

For instance, we don't want to make `position` mandatory because then we would be precluded from testing a scenario where an aircraft lost position information but was still able to report operational status.  Do you have suggestions on what other fields should be mandatory?  Remember, mandatory here means that uss_qualifier must always inform the USS's virtual user being puppeteered of the value of that field in every test ever conducted.